### PR TITLE
[David Ikanji] Sage-A-Solidity-Smart-Contract-For-Auctioning-Flowers-On-The-Celo-Blockchain

### PR DESCRIPTION
**Pull request by Ikanji201**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/03-15-2023-Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain/Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain.md [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+---
+
+![header](../../src/data-tutorials/showcase/intermediate/A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain.png)
+
+
+## INTRODUCTION
+
+**What is Celo?**
+
+  Celo is a decentralized platform with a goal to enable access to financial tools to people worldwide, particularly in developing markets. The platform uses blockchain technology to secure transactions and enable fast, low-cost, and safe transfers of its stablecoin, CELO. The value of CELO is linked to a basket of national currencies to mitigate fluctuations. The Celo network is maintained by a group of validators who operate the blockchain software and are incentivized with CELO tokens. Additionally, the platform comprises several decentralized applications (dApps) that offer financial services, including remittances, loans, and payments.
```
Comment (4undRaiser) [inline blog/03-15-2023-Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain/Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain.md:21]: make sure all text begins in the same line. do the same fixes in  other places too

File: blog/03-15-2023-Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain/Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain.md [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+            _image,
+            _price,
+            0,
+            address(0),
+            _isSale,
+            false
+        );
+        flowersLength++;
+    }
+
```
Comment (4undRaiser) [inline blog/03-15-2023-Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain/Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain.md:157]: line (157 and 158) remove the two extra spacing at the end of the code. do the same fixes in other place where it occurs.

File: blog/03-15-2023-Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain/Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain.md (hunk 1 above)
Comment (Ikanji201) [reply blog/03-15-2023-Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain/Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain.md]: done

File: blog/03-15-2023-Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain/Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain.md (hunk 2 above)
Comment (Ikanji201) [reply blog/03-15-2023-Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain/Sage-Sage-A-Solidity-Smart-Contract-for-Auctioning-Flowers-on-the-celo-Blockchain.md]: done

**Review by developerkunal — APPROVED** [review]
Approved @Ishanpathak1

**Review by Ishanpathak1 — APPROVED** [review]
Approved

**Pull request merged**